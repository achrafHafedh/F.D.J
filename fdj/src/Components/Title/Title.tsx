import { FunctionComponent } from "react";

import { TitleProps } from "./Types";

import "./Title.scss";

const Title: FunctionComponent<TitleProps> = ({ text }) => {
  return <div className="title">{text}</div>;
};

export default Title;
