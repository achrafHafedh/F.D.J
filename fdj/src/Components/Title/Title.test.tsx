import { render } from "@testing-library/react";

import Title from "./Title";

describe("<Title/>", () => {
  const props = {
    text: "Title",
  };
  it("should render Title component", () => {
    const { container } = render(<Title {...props} />);
    expect(container).toMatchSnapshot();
  });
});
