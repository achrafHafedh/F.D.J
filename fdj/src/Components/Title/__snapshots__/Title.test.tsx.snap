// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title/> should render Title component 1`] = `
<div>
  <div
    class="title"
  >
    Title
  </div>
</div>
`;
